esbuild.mjs

```diff
@@ -21,7 +21,7 @@ async function main(tsconfigRaw, entryPoint, platform, format, plugins = []) {
         sourcemap: !production ? "linked" : false,
         platform,
         target: platform === "node" ? "node20" : "es2022",
-        external: ["vscode", "fs", "path", "os"],
+        external: ["child_process", "crypto", "fs", "http", "https", "node:*", "os", "path", "vscode"],
```

Comment:
Seems like the "node:*" item makes most of the others (child_process, crypto, fs, http, https, os, path) redundant?

Reply:
From esbuilds point of view it thinks "node:http" and "http" are different things.  I will re-test in case I can remove some now.